hypervisor/arch/x86/guest/vmexit.c

```diff
@@ -271,6 +272,42 @@ static int32_t unhandled_vmexit_handler(struct acrn_vcpu *vcpu)
 	return 0;
 }
 
+static void vcpu_complete_split_lock_emulation(struct acrn_vcpu *cur_vcpu)
```

Comment:
This is to move from virq.c to here?  Neither file is not perfect to hold this API.  How about to move all the split-lock related APIs into a new file?

Reply:
add "hv: move split-lock logic into dedicated file" to move into new files.